go.mod

```diff
@@ -11,51 +11,44 @@ require (
 	github.com/kr/text v0.2.0
 	github.com/markbates/inflect v1.0.4
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.2.2
-	github.com/onsi/ginkgo/v2 v2.2.0
-	github.com/onsi/gomega v1.20.2
-	github.com/operator-framework/api v0.17.4-0.20221221181915-f1b729684854
-	github.com/operator-framework/helm-operator-plugins v0.0.12-0.20230109213218-ebfbea851192
-	github.com/operator-framework/java-operator-plugins v0.7.1-0.20221007075838-2e24140314fb
-	github.com/operator-framework/operator-lib v0.11.1-0.20220921174810-791cc547e6c5
-	github.com/operator-framework/operator-manifest-tools v0.2.3-0.20230213223805-9262eb48b716
-	github.com/operator-framework/operator-registry v1.26.3-0.20220930210947-614d6a955dc0
-	github.com/prometheus/client_golang v1.13.0
-	github.com/prometheus/client_model v0.2.0
+	github.com/onsi/ginkgo/v2 v2.7.0
+	github.com/onsi/gomega v1.24.2
+	github.com/operator-framework/api v0.17.4-0.20230223191600-0131a6301e42
+	github.com/operator-framework/helm-operator-plugins v0.0.12-0.20230307164205-6e30bde28688
+	github.com/operator-framework/java-operator-plugins v0.7.1-0.20230306190439-0eed476d2b75
+	github.com/operator-framework/operator-lib v0.11.1-0.20230306195046-28cadc6b6055
+	github.com/operator-framework/operator-manifest-tools v0.2.3-0.20230227155221-caa8b9e1ab12
+	github.com/operator-framework/operator-registry v1.26.5-0.20230307160312-3bfcb23fa7d0
+	github.com/prometheus/client_golang v1.14.0
+	github.com/prometheus/client_model v0.3.0
 	github.com/sergi/go-diff v1.2.0
 	github.com/sirupsen/logrus v1.9.0
-	github.com/spf13/afero v1.9.2
-	github.com/spf13/cobra v1.6.0
+	github.com/spf13/afero v1.9.3
+	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.10.0
-	github.com/stretchr/testify v1.8.0
+	github.com/stretchr/testify v1.8.1
 	github.com/thoas/go-funk v0.8.0
-	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
-	golang.org/x/text v0.3.8
-	golang.org/x/tools v0.1.12
+	golang.org/x/mod v0.7.0
+	golang.org/x/text v0.7.0
+	golang.org/x/tools v0.5.0
 	gomodules.xyz/jsonpatch/v3 v3.0.1
 	helm.sh/helm/v3 v3.10.3
-	k8s.io/api v0.25.3
-	k8s.io/apiextensions-apiserver v0.25.3
-	k8s.io/apimachinery v0.25.3
-	k8s.io/cli-runtime v0.25.3
-	k8s.io/client-go v0.25.3
-	k8s.io/kubectl v0.25.3
-	k8s.io/utils v0.0.0-20221012122500-cfd413dd9e85
-	sigs.k8s.io/controller-runtime v0.13.0
-	sigs.k8s.io/controller-tools v0.10.0
-	sigs.k8s.io/kubebuilder/v3 v3.7.1-0.20221011212440-eff842a46496
+	k8s.io/api v0.26.2
+	k8s.io/apiextensions-apiserver v0.26.2
+	k8s.io/apimachinery v0.26.2
+	k8s.io/cli-runtime v0.26.2
+	k8s.io/client-go v0.26.2
+	k8s.io/kubectl v0.26.1
+	k8s.io/utils v0.0.0-20230220204549-a5ecb0141aa5
+	sigs.k8s.io/controller-runtime v0.14.5
+	sigs.k8s.io/controller-tools v0.11.3
+	sigs.k8s.io/kubebuilder/v3 v3.9.0
```

Comment:
There was a [bug](https://kubernetes.slack.com/archives/CAR30FCJZ/p1678204565778959) found in kubebuilder recently that was making it so the go plugins weren't able to finish scaffolding properly. @varshaprasad96 merged https://github.com/kubernetes-sigs/kubebuilder/pull/3270 which should fix this. Since it would be ideal to not pull this bug in I think we should base our kubebuilder dep on the commit with the fix (`3cb60ab19419b1f045221ee8049d835fbb21bf60`)

Reply:
Updated KB to latest commit: `3cb60ab19419b1f045221ee8049d835fbb21bf60`